src/main.rs

```diff
@@ -482,13 +482,13 @@ async fn start_ldk() {
 	let event_notifier = event_ntfn_sender.clone();
 	let listening_port = args.ldk_peer_listening_port;
 	tokio::spawn(async move {
-		let listener = std::net::TcpListener::bind(format!("0.0.0.0:{}", listening_port)).unwrap();
+		let listener = tokio::net::TcpListener::bind(format!("0.0.0.0:{}", listening_port)).await.unwrap();
 		loop {
 			let peer_mgr = peer_manager_connection_handler.clone();
 			let notifier = event_notifier.clone();
-			let tcp_stream = listener.accept().unwrap().0;
+			let tcp_stream = listener.accept().await.unwrap().0;
```

Comment:
Could you remove this `unwrap`? I've just been informed that this can panic if other things are using the same port